server.js

```diff
@@ -2600,15 +2600,29 @@ cache(function(data, match, sendBadge, request) {
 }));
 
 // GitHub release-download-count integration.
-camp.route(/^\/github\/downloads\/([^\/]+)\/([^\/]+)\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+camp.route(/^\/github\/downloads\/([^\/]+)\/([^\/]+)(\/[^\/]+)?\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
   var user = match[1];  // eg, qubyte/rubidium
   var repo = match[2];
-  var tag = match[3];
+
+  var tag = match[3]; //null for all releases
   var asset_name = match[4].toLowerCase(); // eg. total, atom-amd64.deb, atom.x86_64.rpm
   var format = match[5];
-  var release_path = tag !== 'latest' ? 'tags/' + match[3] : 'latest';
-  var apiUrl = 'https://api.github.com/repos/' + user + '/' + repo + '/releases/' + release_path;
+
+  if (tag && (tag.indexOf('/') !== -1)) {
+    tag = tag.split('/').join('');
+  }
```

Comment:
Why can't we replace that `if (…) {…}` by a simple `if (tag) { tag = tag.slice(1); }`?

I was almost tempted to merge and do that change myself, but I may be missing something.